src/engraving/libmscore/mscore.h

```diff
@@ -135,6 +135,9 @@ static const int VISUAL_STRING_NONE      = -100;      // no ordinal for the visu
 static const int STRING_NONE             = -1;        // no ordinal for a physical string (0 = topmost in instrument)
 static const int FRET_NONE               = -1;        // no ordinal for a fret
 
+using ID = uint64_t;
+static constexpr ID INVALID_ID = 0;
```

Comment:
What sort of objects will have these sorts of IDs?

Reply:
I don't want to create dependency on the framework in libmscore because libmscore can be used independently in other projects. Probably later we'll move some types (like ID or async primitives) to libmscore, but now isn't the right time for that :)